src/main/scala/karazin/scala/users/group/week1/homework/Homework.scala

```diff
@@ -40,27 +43,85 @@ object Homework :
 
   object `Boolean Operators` :
 
-    def not(b: Boolean): Boolean = ???
+    def not(b: Boolean): Boolean = {

```

Comment:
1. You can get rid of curly braces after `=`.
2. You should use `;` in the end of lines. Remove them.
3. `b` is already a boolean value, no need to compare it with true or false values.